vscode-extension/package.json

```diff
@@ -15,6 +15,7 @@
     "Other"
   ],
   "main": "./out/extension.js",
+  "activationEvents": ["*"],
```

Comment:
Why is this needed? Also, is there a reason for `*` vs `onStartupFinished`?

Reply:
Oh yea this was just for testing, removing